run_bd_cases.py

```diff
@@ -0,0 +1,11 @@
+
+
+from src.cases import BeamdynCase
+from src.beamdyn_driver import BeamdynDriver
+
+# build the case
+driver = BeamdynDriver("/Users/rmudafor/Development/openfast/build/modules/beamdyn/beamdyn_driver")
```

Comment:
This is the path I don't have... I can find the beamdyn folder directly from the opencast/modules directory though